modules/runner/src/integrationTest/java/org/apache/ignite/internal/benchmark/AbstractMultiNodeBenchmark.java

```diff
@@ -71,7 +71,7 @@ public class AbstractMultiNodeBenchmark {
     protected static Ignite publicIgnite;
     protected static IgniteImpl igniteImpl;
 
-    @Param({"false", "true"})
+    @Param({"false"})
```

Comment:
Please return it back :)

Reply:
Unnecessary changes rolled back, `idToMemberMap` update logic fixed.